circle.yml

```diff
@@ -20,6 +20,9 @@ dependencies:
     # We install and test with all the supported MongoDB versions
     - case $CIRCLE_NODE_INDEX in [1,2]) sudo service mongodb stop ; MONGODB_VERSION=3.2.12 sudo -E .circle/install-and-run-mongodb.sh > /tmp/mongodb-install.log 2>&1 ;; [3,4]) sudo service mongodb stop ; MONGODB_VERSION=3.4.2 sudo -E .circle/install-and-run-mongodb.sh > /tmp/mongodb-install.log 2>&1 ;; esac:
         background: true
+    # We sleep a bit to wait for the background process to start and script to
+    # finish
+    - case $CIRCLE_NODE_INDEX in [1,2]) sleep 10 ;; esac
```

Comment:
Sadly no because it runs in a background (background: true) and that's why we need it to be a separate step.